code/notebook/src/cleaning.py

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
+import re
+import nltk
+from nltk.corpus import stopwords
+from nltk.stem import PorterStemmer, WordNetLemmatizer
+from nltk.tokenize import word_tokenize
+from nltk import pos_tag
+nltk.download('punkt')
+nltk.download('stopwords')
+nltk.download('wordnet')
+nltk.download('averaged_perceptron_tagger')
+
+stemmer = PorterStemmer()
+lemmatizer = WordNetLemmatizer()
+
+def nlp_cleaning(text):
```

Comment:
Add the code for removing special character as well, the title contains some special characters in them.
all the non ASCII values should be removed, refer to the image below

in the title

![image](https://github.com/user-attachments/assets/df9b042e-9379-485b-808e-05215d676d71)